src/core/pdf-parser/parseBool.ts

```diff
@@ -18,21 +18,23 @@ const parseBool = (
   input: Uint8Array,
   { onParseBool }: IParseHandlers = {},
 ): [PDFBoolean, Uint8Array] | void => {
-  const boolRegex = /^(?:[\n|\r| ]*)(true|false)((?= |\]|\n|\r))?/;
+  const trimmed = trimArray(input);
+  const boolRegex = /^(?:[\0\t\n\f\r ]*)(true|false)((?=[\0\t\n\f\r \]]))?/;
 
   // Search for first character that isn't part of a boolean
   let idx = 0;
-  while (String.fromCharCode(input[idx]).match(/^[ \n\rtruefalse]/)) idx += 1;
+  while (String.fromCharCode(trimmed[idx]).match(/^[\0\t\n\f\r truefalse]/) && idx < trimmed.length) idx += 1;
+  console.log('parseBool', idx, encodeURI(arrayToString(trimmed, 0, 100)));
```

Comment:
```suggestion
```
I assume this was just for debugging purposes?

Reply:
Yes, sorry, shouldn't have been committed.